project-management-infrastructure/src/main/java/life/qbic/projectmanagement/infrastructure/communication/MessageConsumer.java

```diff
@@ -0,0 +1,49 @@
+package life.qbic.projectmanagement.infrastructure.communication;
+
+import static life.qbic.logging.service.LoggerFactory.logger;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import life.qbic.logging.api.Logger;
+import life.qbic.projectmanagement.application.communication.broadcasting.IntegrationEvent;
+import life.qbic.projectmanagement.application.communication.broadcasting.MessageRouter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jms.annotation.JmsListener;
+import org.springframework.stereotype.Component;
+
+/**
+ * <b><class short description - 1 Line!></b>
+ *
+ * <p><More detailed description - When to use, what it solves, etc.></p>
+ *
+ * @since <version tag>
+ */
+@Component
+public class MessageConsumer {
+
+  private static final Logger log = logger(MessageConsumer.class);
+
+  private final MessageRouter messageRouter;
+
+  @Autowired
+  public MessageConsumer(MessageRouter messageRouter) {
+    this.messageRouter = messageRouter;
+    log.info("Build Message Consumer");
+  }
+
+  @JmsListener(destination = "User")
```

Comment:
Is this
`private static final String QUEUE = "User";`
from MessageDispatcher? Any way we can define this in one place only?

Reply:
Well spotted! I think it is possible to ingest this dynamically and configure it in the ``application.properties``